### DISCUSSION TITLE
File Size - CDE 11479876

### DISCUSSION THREAD

**Discussion by GeoffLyle** [CBIIT/ccdi-federation-api#55] [Metadata Field Ratification]
caDSR Link: [11479876](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=11479876%20and%20ver_nr=1)

Definition: The measure (in bytes) of how much spade a data file takes up on a storage medium.

Discussion:  Would we like to also include a human friendly format or just the number of bytes?

**Comment by dunnpa** [comment]
number of bytes please

**Comment by Ssandor13** [comment]
St. Jude will be able to provide size in bytes. 

If there is a CDE that provides a human friendly format and conversion that the Federation can use, we would agree to also implement this as well.

**Comment by GeoffLyle** [comment]
CDE accepted at January 25th meeting.

Field will include an integer value of the file size in bytes.  No CDE currently available for a human readable format.
